src/System Application/App/Azure AD User/src/AzureADGraphUserImpl.Codeunit.al

```diff
@@ -359,6 +372,11 @@ codeunit 9011 "Azure AD Graph User Impl."
         exit(LanguageId);
     end;
 
+    [InternalEvent(false)]
+    local procedure OnIsUserDelegated(var IsDelegatedUser: Boolean; var Handled: Boolean)
```

Comment:
Why do we need an event?
I would skip this event, there should not be need to test this from AL.